tests/Datagrid/DatagridMapperTest.php

```diff
@@ -56,6 +56,7 @@ protected function setUp(): void
     {
         $datagridBuilder = $this->createMock(DatagridBuilderInterface::class);
 
+        /** @var ProxyQueryInterface $proxyQuery */
```

Comment:
Maybe the phpunit plugin for psalm solves this?

Reply:
No, it's an issue about contravariance and generics. It's considered as the right behavior.

I dont want this var to be consideref as ProxyQueryInterface and Mock, but just a ProxyQueryInterface.